Packs/AutoFocus/Integrations/AutoFocusTagsFeed/AutoFocusTagsFeed.py

```diff
@@ -0,0 +1,509 @@
+    Args:
+        client: Client class
+        name: The indicator's name
+        tag_type: The indicator's type
+        related_tags: A list of all indicators related to the spesific indicator
+    Returns:
+        a list represents the relationships of an indicator.
+    """
+    relationships: list = []
+    for related_tag in related_tags:
+        try:
+            related_tag_details = client.get_tag_details(related_tag)
+        except DemistoException:
+            demisto.debug(f'AutoFocus Tags Feed: Could not create relationship for {name} with {related_tag}.')
+            continue
+        if related_tag_details:
+            tag = related_tag_details.get('tag')
+            related_tag_name = tag.get('tag_name')
+            tag_class = tag.get('tag_class')
+            source = tag.get('source')
+            related_tag_type = get_tag_class(tag_class, source)
+            if related_tag_type:
+                relationships.append(
+                    create_relationship(name, tag_type, related_tag_name, related_tag_type).to_indicator())
+    return relationships
+
+
+def create_relationship(a_name: str, a_class: str, b_name: str, b_class: str):
+    return EntityRelationship(
+        name=MAP_RELATIONSHIPS.get(a_class, {}).get(b_class),
+        entity_a=a_name,
+        entity_a_type=a_class,
+        entity_b=b_name,
+        entity_b_type=b_class,
+        reverse_name=MAP_RELATIONSHIPS.get(b_class, {}).get(a_class),
+    )
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """Builds the iterator to check that the feed is accessible.
+    Args:
+        client: Client object.
+    Returns:
+        Outputs.
+    """
+    try:
+        response = client.get_tags(data={'pageSize': 1})
+        tag_list = response.get('tags', [])
+        tag = tag_list[0] if len(tag_list) > 0 else {}
+        tag_name = tag.get('public_tag_name')
+        client.get_tag_details(tag_name)
+    except Exception as e:
+        raise DemistoException(str(e))
+    return 'ok'
+
+
+def fetch_indicators(client: Client,
+                     is_get_command: bool,
+                     tlp_color: Optional[str] = None,
+                     feed_tags: List = None,
+                     limit: int = -1) -> List[Dict]:
+    """Retrieves indicators from the feed
+    Args:
+        is_get_command: is this method called from the get-indicators-command
+        client (Client): Client object with request
+        tlp_color (str): Traffic Light Protocol color
+        feed_tags (list): tags to assign fetched indicators
+        limit (int): limit the results
+    Returns:
+        Indicators list.
+    """
+    iterator = client.build_iterator(is_get_command, limit)
+    indicators = []
+    for tag_details in iterator:
+        tag = tag_details.get('tag')
+        value_ = tag.get('tag_name')
+        tag_class = tag.get('tag_class')
+        source = tag.get('source')
+        type_ = get_tag_class(tag_class, source)
```

Comment:
The names here are a bit confusing. Let's try to make them clearer - 
* "tag" can be "tag_dict" or something of that sort
* "value_" - is this the tag value? if so - "tag_value"
* "type_" - same as value

clear and indicative naming can really help understand the code. Also, do we need default values here?